src/components/DatePicker/DatePicker.tsx

```diff
@@ -63,11 +64,10 @@ export default function DatePicker(props: Props): JSX.Element {
           minDate={props.minDate ? moment(props.minDate) : undefined}
           maxDate={props.maxDate ? moment(props.maxDate) : undefined}
           inputFormat='yyyy-MM-DD'
-          value={props.value ? moment(props.value, 'YYYY-MM-DD').toDate() : null}
+          value={temporalValue}
           onChange={(newValue: any) => {
-            if (newValue?.isValid()) {
-              props.onChange(props.id, newValue?.toDate());
-            }
+            setTemporalValue(newValue);
+            props.onChange(props.id, newValue?.toDate(), newValue.isValid());
```

Comment:
@constanzauanini i made this change so callers of this component can do error handling (like we currently do).

Reply:
i reverted this - let the clients decide if date is valid or not.